ansible/roles/openshift_master_upgrade/tasks/main.yml

```diff
@@ -0,0 +1,40 @@
+---
+# tasks file for openshift_master_upgrade
+#- name: Remove Masters from elbs
+# This is probably best as it's own role since elbs are AWS specific
+
+- name: Stop OpenShift API
+  service: 
+    name: atomic-openshift-master-api
+    state: stopped
+
+- name: Stop OpenShift Controller
+  service: 
+    name: atomic-openshift-master-controllers
+    state: stopped
+
+- name: Ensure that the Master IP address is out of the Kubernetes endpoint #This will only remove the IP from the enpoint in 3.2.1.9 and later
+  pause:
+    seconds: 5
+
+- name: Yum update docker
```

Comment:
Will this actually update anything? I thought we were blocking docker updates in the yum config. I think the second update statement below is probably good enough. (line 33)